plugin/keyring_vault/tests/mtr/innodb_online_alter_encryption.test

```diff
@@ -20,10 +20,7 @@ call mtr.add_suppression("keyring_vault initialization failure");
 call mtr.add_suppression("A message intended for a client cannot be sent there as no client-session is attached");
 --enable_query_log
 
---let $keyring_restart_param = restart: $KEYRING_VAULT_PLUGIN_OPT $KEYRING_VAULT_PLUGIN_EARLY_LOAD --loose-keyring_vault_config=$vault_conf_file --innodb-sort-buffer-size=64k --innodb_encrypt_online_alter_logs=ON
-
---replace_result $MYSQLTEST_VARDIR MYSQLTEST_VARDIR
-eval SET @@global.keyring_vault_config="$vault_conf_file";
+--let $keyring_restart_param = restart: $KEYRING_VAULT_PLUGIN_OPT $KEYRING_VAULT_PLUGIN_EARLY_LOAD --keyring_vault_config=$vault_conf_file --innodb-sort-buffer-size=64k --innodb_encrypt_online_alter_logs=ON
```

Comment:
This test is inherited from PS. How it is that it works for PS?
https://github.com/percona/percona-server/blob/c676f75d020dd0a56716f9dd6c7526c15818cf74/plugin/keyring_vault/tests/mtr/innodb_online_alter_encryption.test#L25

Reply:
Updated the commit message that explains, why the suite works in Percona Server.